tests/test_slogpy_console.py

```diff
@@ -0,0 +1,24 @@
+"""Import slogpy from slog"""
+from slogpy.slog import Slog as slog  # noqa: N813
+
+"""This is a setup function"""
+def setup_function():
+    """set "file_logging = False", so it will not create a log file for output."""
+    slog.initialize(file_logging=False)
+
+"""First unit test"""
+def test_info_console_output(capsys, msg="This is a console test"):
+    """Passing a message to the info method and then capture the message from the console"""
```

Comment:
suggestion: It would be helpful to use the Given-When-Then style of describing what this test does. https://martinfowler.com/bliki/GivenWhenThen.html

ex:
Given: using slogpy
When: log info
Then: message is in stdout

The difference is this shows what is expected, i.e. what the test is testing.  The docstring above is close to that, but it is more describing the how the testing is done.  The code describes the how very well, so the comment is not high value.